ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/Vectorizer.java

```diff
@@ -2411,6 +2411,21 @@ private boolean getOnlyStructObjectInspectors(ReduceWork reduceWork,
     }
   }
 
+  private void setAllowAdditionalUDFs(HiveConf hiveConf) {
+    String[] udfs =
+        HiveConf.getTrimmedStringsVar(hiveConf, HiveConf.ConfVars.HIVE_VECTOR_ADAPTOR_CUSTOM_UDF_WHITELIST);
```

Comment:
Maybe add `GENERIC` to the property name, since you can't register here aggregation UDFs